From: Borislav Petkov <bp@suse.de>
Date: Wed, 16 Aug 2017 19:41:52 +0200
Subject: EDAC, ghes: Remove symbol exports
Git-commit: c9c8b4d6d0259ac914309f8fbcef022792c99a89
Patch-mainline: v4.15-rc1
References: bsc#1042420

They're called from builtin code so no need for the exports.

Signed-off-by: Borislav Petkov <bp@suse.de>
---
 drivers/edac/ghes_edac.c |    3 ---
 1 file changed, 3 deletions(-)

--- a/drivers/edac/ghes_edac.c
+++ b/drivers/edac/ghes_edac.c
@@ -403,7 +403,6 @@ void ghes_edac_report_mem_error(struct g
 	/* Report the error via EDAC API */
 	edac_raw_mc_handle_error(type, mci, e);
 }
-EXPORT_SYMBOL_GPL(ghes_edac_report_mem_error);
 
 int ghes_edac_register(struct ghes *ghes, struct device *dev)
 {
@@ -508,7 +507,6 @@ int ghes_edac_register(struct ghes *ghes
 	mutex_unlock(&ghes_edac_lock);
 	return 0;
 }
-EXPORT_SYMBOL_GPL(ghes_edac_register);
 
 void ghes_edac_unregister(struct ghes *ghes)
 {
@@ -524,4 +522,3 @@ void ghes_edac_unregister(struct ghes *g
 		}
 	}
 }
-EXPORT_SYMBOL_GPL(ghes_edac_unregister);
